### PR TITLE
Serhii Murashko's card

### PR DESCRIPTION
**Pull request by SimpleSquirrelz**
Added my card to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:117]: You should leave a single new line between the template and your card. (That blank line should be line-117, and your card should start from line-118)

File: index.html [hunk 2]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+        <div class="card">
```
Comment (dilshankarunarathne) [inline index.html:118]: Make sure to add a comment on top of your card. 
The top comment from the template 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->` 
should be changed to `<!-- ________ your_name card START ________  -->`

File: index.html [hunk 3]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://learngitbranching.js.org" target="_blank" title="Third resource">
+                  Learn git by playing game
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:146]: Please change this `*END TEMPLATE*` part into your name as well. It would be easier for everyone to identify your card in code.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @SimpleSquirrelz,
make these changes and get back to us, so we can review and merge your pull request.

**Comment by SimpleSquirrelz** [comment]
Added comments and new line :)

**Comment by dilshankarunarathne** [comment]
@SimpleSquirrelz your code looks good. Congratulations ! ❤👍
Your code will be merged to the master branch.

**Pull request merged**